nilearn/plotting/tests/test_baseline_comparisons.py

```diff
@@ -301,6 +302,43 @@ def test_plot_surf_surface(plot_func, view, hemi):
     )
 
 
+@pytest.mark.skipif(
+    not (is_plotly_installed() and is_kaleido_installed),
```

Comment:
```suggestion
    not (is_plotly_installed() and is_kaleido_installed()),
```

`is_kaleido_installed` is a function, right?